modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -6,53 +6,236 @@
+- An external Identity Provider connected to the same LDAP/AD user backend and configured to work with the ownCloud components
+- The https://marketplace.owncloud.com/apps/openidconnect[OpenID Connect App] installed on ownCloud Server
+- Configuration for the OpenID Connect App in `config.php` on ownCloud Server
+- Service discovery for the ownCloud Clients
+
+The below sections will explain these areas and provide configuration examples using https://github.com/Kopano-dev/konnect[Kopano Konnect] as the external Identity Provider.
+
+For the configuration examples, let's assume we have
+- ownCloud Server available as `https://cloud.example.com`
+- Kopano Konnect available as `https://idp.example.com`
+
+=== Set up and configure the Identity Provider
+
+In order to get your Identity Provider running and ready to be used with ownCloud, you have to obtain Kopano Konnect and run it with a set of configuration values which can be provided as environment variables. See the below sections for details.
+
+Specifically, you have to
+- provide basic configuration
+- connect Kopano Konnect to LDAP/AD
+- set up a reverse proxy to expose the routes required to connect to Kopano Konnect. You'll find instructions in the https://documentation.kopano.io/kopanocore_administrator_manual/configure_kc_components.html#configure-a-webserver-for-konnect[Kopano Konnect documentation].
+
+TIP: Kopano Konnect can be set up via Docker. Images are available on Docker Hub (`kopano/kopano_konnect`).
+TIP: For general information on Kopano Konnect, please refer to the https://documentation.kopano.io/kopanocore_administrator_manual/configure_kc_components.html#configure-kopano-konnect[Kopano Documentation]. The configuration below is an example to make Kopano Konnect work with ownCloud Server.
+
+==== Basic configuration
+
+To run Kopano Konnect, some basic configuration is necessary. Please replace the below environment variables with the paths and values suited for your environment.
+
+```
+- FQDN=idp.example.com
+- allow_client_guests=yes
+- allow_dynamic_client_registration=yes
+- eckey=/kopano/ssl/meet-kwmserver.pem
+- ecparam=/kopano/ssl/ecparam.pem
+- encryption_secret_key=/kopano/ssl/konnectd-encryption.key
+- identifier_registration_conf=/kopano/ssl/konnectd-identifier-registration.yaml
+- identifier_scopes_conf=/etc/kopano/konnectd-identifier-scopes.yaml
+- signing_private_key=/kopano/ssl/konnectd-tokens-signing-key.pem
+- validation_keys_path=/kopano/ssl/konnectkeys
+```
+
+==== Connect Kopano Konnect to LDAP/AD
+
+To allow Kopano Konnect to authenticate ownCloud users it is necessary to connect it to the same LDAP/AD server which is used by the ownCloud Server. Below you find the environment variables needed to connect Kopano Konnect to an LDAP/AD server. Please replace them with the values suited for your environment (e.g., from the ownCloud LDAP configuration).
```

Comment:
you describe on how to setup kopano but not the ldap.
especially the config below highly depends on the ldap setup.

this is inconsistent and will result in not working examples -> bad quality and bad ux - sorry